src/main/java/com/salesforce/emp/connector/example/DevLoginExample.java

```diff
@@ -59,7 +59,7 @@ public static void main(String[] argv) throws Throwable {
 
         connector.start().get(5, TimeUnit.SECONDS);
 
-        long replayFrom = EmpConnector.REPLAY_FROM_TIP;
+        long replayFrom = EmpConnector.REPLAY_FROM_EARLIEST;
```

Comment:
Why change this?

Reply:
To make it consistent with LoginExample and BearerTokenExample which use the REPLAY_FROM_EARLIEST option each.

Reply:
Ah got it, thanks. 👍